etc/default_config.env

```diff
@@ -15,13 +17,30 @@ BENTO_CBIOPORTAL_ENABLED=false
 BENTO_GIT_CLONE_HTTPS=0
 
 # Prefix for standardized container names
-#  - non-defaault values allow multiple deployments on the same machine.
+#  - non-default values allow multiple deployments on the same machine.
 BENTOV2_PREFIX=bentov2
 
+# Root data directory on host. Bind volumes for services will be put
+# here as subdirectories and mounted into the relevant containers.
+BENTOV2_ROOT_DATA_DIR=~/bentov2/data
+
 # Gateway/domains -----------------------------------------------------
 BENTOV2_DOMAIN=bentov2.local
 BENTOV2_PORTAL_DOMAIN=portal.${BENTOV2_DOMAIN}
 BENTOV2_AUTH_DOMAIN=bentov2auth.local
+# Unused if cBioPortal is disabled:
+BENTOV2_CBIOPORTAL_DOMAIN=cbioportal.${BENTOV2_DOMAIN}
+# ---------------------------------------------------------------------
+
+# Certificates --------------------------------------------------------
+BENTOV2_AUTH_FULLCHAIN_RELATIVE_PATH=/auth_fullchain1.crt
+BENTOV2_AUTH_PRIVKEY_RELATIVE_PATH=/auth_privkey1.key
+BENTOV2_GATEWAY_INTERNAL_PORTAL_FULLCHAIN_RELATIVE_PATH=/portal_fullchain1.crt
+BENTOV2_GATEWAY_INTERNAL_PORTAL_PRIVKEY_RELATIVE_PATH=/portal_privkey1.key
+BENTOV2_GATEWAY_INTERNAL_FULLCHAIN_RELATIVE_PATH=/fullchain1.crt
+BENTOV2_GATEWAY_INTERNAL_PRIVKEY_RELATIVE_PATH=/privkey1.key
+BENTOV2_GATEWAY_INTERNAL_CBIOPORTAL_FULLCHAIN_RELATIVE_PATH=/cbioportal_fullchain1.crt
```

Comment:
Even with `BENTOV2_PRIVATE_MODE=false` in local.env, the gateway is looking for the cbioportal certs, which are not generated due to `BENTOV2_PRIVATE_MODE` being false. This causes the gateway to fail on start.

Reply:
do you mean with cbioportal turned off? im not sure how private mode would be related

Reply:
Following tests done with `BENTOV2_PRIVATE_MODE=true`, the gateway is able to start and the auth setup can be completed. However, the following services fail on start when spinning the whole cluster:
- bentov2-gohan-elasticsearch: `AccessDeniedException: /usr/share/elasticsearch/data/nodes`
- bentov2-cbioportal-db: need to set MYSQL_ROOT_PASSWORD, MYSQL_ALLOW_EMPTY_PASSWORD, MYSQL_RANDOM_ROOT_PASSWORD

Reply:
Oh yeah sorry bad copy paste on my end

Reply:
Added the missing cbio vars to etc/bento_dev.env, now the only exited service container is `bentov2-gohan-elasticsearch`

Reply:
Quick fix for bentov2-gohan-elasticsearch, run `chmod 777 ./data/gohan/elasticsearch` and then it should start. This has been known for a while now I think so I will resolve